src/views/Home/index.tsx

```diff
@@ -17,15 +17,28 @@ import WelcomeText from './WelcomeText';
 import './index.scss';
 
 const Banners = () => {
+  const history = useHistory();
   const location = useLocation<any>();
+  const [confirmationText, setIsConfirmationText] = useState('');
+
+  useEffect(() => {
```

Comment:
On page mount, if the text exists in the routing, we store that text then clear it from the router. Clearing it from the router makes sure the text isn't shown again once the local state is cleared.